### PR TITLE
Fix tables on gitbook

### PR DESCRIPTION
**Pull request by TheDauntless**
Two pages have broken tables on GitHub, most likely due to the pagebreak html. I've removed them in this branch.

https://mobile-security.gitbook.io/masvs/security-requirements/0x07-v2-data_storage_and_privacy_requirements
https://mobile-security.gitbook.io/masvs/security-requirements/0x15-v8-resiliency_against_reverse_engineering_requirements

### CODE REVIEW

**Comment by TheDauntless** [comment]
And maybe we need to figure out why they were there, and if something else would break now.

**Comment by OMTGreleaser** [comment]
Results for commit 5b2217f95b66b705fc1f16cf77c8434fdf576e14: Broken link result: 0 .
  
 Markdown result errorlength: 2 .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: Document/0x07-V2-Data_Storage_and_Privacy_requirements.md [hunk 1]
```diff
@@ -14,7 +14,6 @@ Sensitive data in the context of the MASVS pertains to both user credentials and
 - Highly sensitive data that would lead to reputational harm and/or financial costs if compromised: Contractual information, information covered by non-disclosure agreements, management information;
 - Any data that must be protected by law or for compliance reasons.
 
-<div style="page-break-after: always;"></div>
```
Comment (commjoen) [inline Document/0x07-V2-Data_Storage_and_Privacy_requirements.md]: Can we try:
```html
<div style="page-break-after: always;">
\pagebreak
</div>```
instead?

Comment (commjoen) [reply Document/0x07-V2-Data_Storage_and_Privacy_requirements.md]:
```suggestion
<div style="page-break-after: always;">
\pagebreak
</div>
```

File: Document/0x15-V8-Resiliency_Against_Reverse_Engineering_Requirements.md [hunk 2]
```diff
@@ -18,7 +18,6 @@ The following considerations apply:
 
 3. The effectiveness of the protection should always be verified by a human expert with experience in testing the particular types of anti-tampering and obfuscation used (see also the "reverse engineering" and "assessing software protections" chapters in the Mobile Security Testing Guide).
 
-<div style="page-break-after: always;"></div>
```
Comment (commjoen) [inline Document/0x15-V8-Resiliency_Against_Reverse_Engineering_Requirements.md]: see above ;)

Comment (commjoen) [reply Document/0x15-V8-Resiliency_Against_Reverse_Engineering_Requirements.md]: let's try 
```suggestion
<div style="page-break-after: always;">
\pagebreak
</div>

```

**Pull request merged**